nvflare/apis/impl/job_def_manager.py

```diff
@@ -87,15 +88,12 @@ def filter_job(self, meta: dict):
 
 
 class SimpleJobDefManager(JobDefManagerSpec):
-    def __init__(self, uri_root: str = "jobs", job_store_id: str = "job_store", temp_dir: str = "/tmp"):
+    def __init__(self, uri_root: str = "jobs", job_store_id: str = "job_store"):
```

Comment:
I saw the fed_server.json in poc being updated.  However this change could break provisioned fed_server.json.

Reply:
I think this variable is not needed I use `temp_dir = tempfile.mkdtemp()` to replace this.

Since it is 2.1, should we just re-provision?

Reply:
The sample project.yml, included in nvflare, has temp_dir in job_manager component.  That one will need updated, too.
```          - id: job_manager    # This id is reserved by system.  Do not change it.
            path: nvflare.apis.impl.job_def_manager.SimpleJobDefManager
            args:
              uri_root: /tmp/jobs-storage
              job_store_id: job_store
              temp_dir: /tmp/jobs_temp
```

Reply:
The sample project.yml, included in nvflare, has temp_dir in job_manager component.  That one will need updated, too.
```          - id: job_manager    # This id is reserved by system.  Do not change it.
            path: nvflare.apis.impl.job_def_manager.SimpleJobDefManager
            args:
              uri_root: /tmp/jobs-storage
              job_store_id: job_store
              temp_dir: /tmp/jobs_temp
```

Reply:
fixed